pages/recipes.html

```diff
@@ -0,0 +1,261 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Recipes</title>
+    <link rel="stylesheet" href="../styles/common.css">
+    <link rel="stylesheet" href="../styles/meals.css">
```

Comment:
meals.css shouldn't be here.

If you have some css from meals that should be here, it only means that you have to move that css to common.css

Reply:
All stylings that are identical between html files have now been placed in the common.css file and duplicate styling from separate .css files has been removed.